src/components/app/header/index.tsx

```diff
@@ -345,3 +346,155 @@ const IconX = () => (
     </g>
   </svg>
 )
+
+const FlyoutMenu = () => {
+  const browse = [
+    {
+      name: 'React',
+      href: '/q/react',
+    },
+    {
+      name: 'JavaScript',
+      href: '/q/javascript',
+    },
+    {name: 'Angular', href: '/q/angular'},
+    {
+      name: 'CSS',
+      href: '/q/css',
+    },
+    {
+      name: 'TypeScript',
+      href: '/q/typescript',
+    },
+    {
+      name: 'AWS',
+      href: '/q/aws',
+    },
+    {
+      name: 'Node.js',
+      href: '/q/node',
+    },
+    {
+      name: 'Next.js',
+      href: '/q/next',
+    },
+    {
+      name: 'Docker',
+      href: '/q/docker',
+    },
+    {
+      name: 'Vue.js',
+      href: '/q/vue',
+    },
+    {
+      name: 'ReactNative',
+      href: '/q/react-native',
+    },
+    {
+      name: 'Algolia',
+      href: '/q/algolia',
+    },
+    {
+      name: 'Python',
+      href: '/q/python',
+    },
+    {
+      name: 'Go',
+      href: '/q/go',
+    },
+  ]
+  const callsToAction = [
```

Comment:
`callsToAction` doesn't feel like quite the right name here. Something like `contentTypes` or `contentTypeLinks` would be more accurate. Small nitpick, but clear naming helps keep this readable.

Reply:
Change done! I have renamed it as `contentSectionLinks`